picard/ui/itemviews.py

```diff
@@ -467,9 +468,21 @@ def drop_urls(urls, target):
         files = []
         new_files = []
         for url in urls:
+            log.debug("Dropped the URL: %r", url.toString(QtCore.QUrl.RemoveUserInfo))
             if url.scheme() == "file" or not url.scheme():
-                # Dropping a file from iTunes gives a filename with a NULL terminator
-                filename = os.path.normpath(os.path.realpath(unicode(url.toLocalFile()).rstrip("\0")))
+                # Workaround for https://bugreports.qt.io/browse/QTBUG-40449
+                # OSX Urls follow the NSURL scheme and need to be converted
+                if sys.platform == 'darwin' and unicode(url.path()).startswith('/.file/id='):
```

Comment:
I would move the platform test and try/import stuff outside the main loop, it can be done once at module level, just use a flag and test for it here. We may need the same kind of workaround elsewhere.